charts/target-group-binding/README.md

```diff
@@ -0,0 +1,48 @@
+# aws-target-group-binding
+
+A Helm chart for deploying a TargetGroupBinding for use with the AWS LoadBalancer Controller
+
+## TL;DR
+
+```bash
+helm repo add snowplow-devops https://snowplow-devops.github.io/helm-charts
+helm install aws-target-group-binding snowplow-devops/aws-target-group-binding
+```
+
+## Introduction
+
+This chart a TargetGroupBinding resource that can be combined with an externally provisioned AWS Loadbalancer
```

Comment:
This sentences feels like its missing a word somewhere.  "This chart a TargetGroupBinding" doesn't read well to me.

Reply:
reworded